guides/doc-Provisioning_Guide/images/ProvisioningUserData-Sequence.txt

```diff
@@ -0,0 +1,16 @@
+title VMware Template Provisioning
+participant User
+participant Satellite
+participant VCenter
+participant Capsule X
+
+participant VM
+
+User --> Satellite:Provision VM in network X
+Satellite --> VCenter:Create a VM from template (port 443)
+VCenter --> VM: Clones VM from template
+VM --> Capsule X: Call Back to Sat port 80 via Capsule X 8000
+Capsule X --> Satellite: forward Userdata on 80
+Satellite --> Satellite: redirect to 443
```

Comment:
```suggestion
activate Satellite
Satellite --> Satellite: redirect to 443
deactivate Satellite
```
Add a box to make it clear what redirects where.

Reply:
If you think this does not work well, we can change the wording to make it more clear what redirects where to `Redirect Userdata to port 443`